vendor/engines/secure_rooms/db/migrate/20170220203130_add_indala_number_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddIndalaNumberToUser < ActiveRecord::Migration
+  def change
```

Comment:
Code climate is flagging missing lines at the beginning/end of the class. I don't care so much in migrations, but I also think a happy rubocop is a benign rubcop.